griptape/utils/structure_visualizer.py

```diff
@@ -41,12 +41,15 @@ def __render_tasks(self, tasks: list[BaseTask]) -> str:
 
     def __render_task(self, task: BaseTask) -> str:
         from griptape.drivers import LocalStructureRunDriver
-        from griptape.tasks import StructureRunTask
+        from griptape.tasks import BranchTask, StructureRunTask
 
         parts = []
         if task.children:
             children = " & ".join([f"{self.build_node_id(child)}" for child in task.children])
-            parts.append(f"{self.build_node_id(task)}--> {children};")
+            if isinstance(task, BranchTask):
+                parts.append(f"{self.build_node_id(task)}{{ {self.build_node_id(task)} }}--> {children};")
```

Comment:
can we add the nice [dotted line like this](https://github.com/griptape-ai/griptape/pull/913/files#diff-bebfb804f6ed2a4f064203cffa668c2b0638e75060c386a7a5bfba3f3384c11cR46)?